wolfssl/wolfcrypt/settings.h

```diff
@@ -203,6 +203,9 @@
 /* Uncomment next line if using Espressif ESP32-WROOM-32SE */
 /* #define WOLFSSL_ESPWROOM32SE */
 
+/* Uncomment next line if using Espressif ESP8266 */
+/* #define WOLFSSL_ESP8266 */
```

Comment:
Instead of adding here I'd prefer a followup PR in the documentation repo. This method for uncommenting a line in settings.h is rarely used anymore.

Reply:
Perhaps "in addition to adding here"?  The documentation currently states: "_For necessary defines, see the `WOLFSSL_ESPWROOM32` define in `settings.h`_"; Given your comment, I assume there's a "--flag" someplace to turn these features on and off?

Reply:
sounds good. I added some [notice](https://github.com/gojimmypi/wolfssl/blob/83f4f9e5ef43e14722e2456be6def7d7098f1aab/wolfssl/wolfcrypt/settings.h#L22) text indicating the preferred method, and moved the ESP8266 define to the [./IDE/Espressif/ESP-IDF/user_settings.h](https://github.com/gojimmypi/wolfssl/blob/83f4f9e5ef43e14722e2456be6def7d7098f1aab/IDE/Espressif/ESP-IDF/user_settings.h#L25)

by far the most difficulty I had in getting started was the various settings / files / methods / locations.

Reply:
oh, and the docs seems to me failing at the moment:

![image](https://user-images.githubusercontent.com/13059545/165559182-956b2509-0ab2-4ac7-90c7-8e4ce3c299e1.png)